Composer/packages/electron-server/electron-builder-config.json

```diff
@@ -56,6 +57,8 @@
     "runAfterFinish": true,
     "installerIcon": "resources/composerIcon.ico",
     "uninstallerIcon": "resources/composerIcon.ico",
```

Comment:
can we remove `resources/` now since we used it above as a directory?

Reply:
good catch! I think we can